public/js/map.js

```diff
@@ -133,7 +133,7 @@ function initAutocomplete() {
         function addFoodSanitationData() {
             $.ajax({
                 method: 'GET',
-                url: 'http://hampton.code4hr.org/food/sanitation',
+                url: 'http://hercules.code4hr.org/food/sanitation',
```

Comment:
Shame on Alicia for changing my comment and extra shame for such a poor mistake.